src/ts/misc.ts

```diff
@@ -378,6 +379,16 @@ export function showContentPlainText(el: HTMLElement): void {
     return response.text();
   }).then(fileContent => {
     (document.getElementById('plainTextArea') as HTMLTextAreaElement).value = fileContent;
+    const mdPreviewWrapper = document.getElementById('mdPreviewWrapper') as HTMLDivElement;
```

Comment:
why is there a need for a wrapper?

I'm sure the code can be further simplified.

Reply:
There is a wrapper because [there are two columns](https://github.com/elabftw/elabftw/pull/4042/files#diff-f59bb09e5c4c6ed71f46f835a604cab55638c1550ffca286a07343a1c14e7dd3R11-R20). I could have named it `mdPreviewColumn`. The mdPreviewWrapper column has an id because it is only visible if we have md. For txt files it is hidden.
This seems to be [the usual way how columns are done with bootstrap](https://getbootstrap.com/docs/4.6/layout/grid/#how-it-works)?
Again, this could also be achieved by css selectors but it seems to be less efficient to me.